### DISCUSSION TITLE
Does MacOS Ventura Break Mac Mouse Fix?

### DISCUSSION THREAD

**Discussion by coffinmilk** [noah-nuebling/mac-mouse-fix#319] [Questions] [answered]
I'm unable to turn on Mac Mouse Fix in Ventura, is anyone else experiencing the same issue?

**Comment by brainsplurge** [comment]
I'm experiencing issues with this too - it turns on but none of the changes work.

**Comment by nianiam** [comment]
Just a heads up, it seems like there are bugs in the beta versions of Ventura which are causing this problem. The author seems to be waiting to see if the release version will fix this. 

https://github.com/noah-nuebling/mac-mouse-fix/issues/297

**Comment by noah-nuebling** [comment]
Hi guys! The [latest beta version](https://github.com/noah-nuebling/mac-mouse-fix/releases/tag/2.2.1-Beta-1) should work great on Ventura.

**Reply by nthused** [reply]
Works great! Thank you Noah!

**Comment by noah-nuebling** [comment, marked as answer]
Hey guys! Marking this as answered now, assuming that [2.2.1 Beta 3](https://github.com/noah-nuebling/mac-mouse-fix/releases/tag/2.2.1-Beta-3) fixes all these issues. Let me know if you have more problems!
